sujit/school/src/database/clientdb.ts

```diff
@@ -0,0 +1,41 @@
+const { Pool } = require("pg")
+const config = require("../config/config")
+
+export const pool = new Pool({
+    user: config.dbUser,
+    database: config.dbName,
+    host: config.dbHost,
+    port: config.dbPort,
+    password: config.dbPassword,
+    max: 20,
+    connectionTimeoutMillsis: 0,
+    idleTimeoutMillsis: 0
+})
+
+pool.on('error', function (error: any, client: any) {
+
+    console.log("postgressql error event")
```

Comment:
We should be throwing error if connection fails. Because application can no longer work if db connection fails.